nemo/collections/nlp/nm/trainables/common/huggingface/bert_nm.py

```diff
@@ -160,5 +160,12 @@ def list_pretrained_models() -> Optional[List[PretrainedModelInfo]]:
             pretrained_models.append(model_info)
         return pretrained_models
 
-    def forward(self, input_ids, token_type_ids, attention_mask):
+    def forward(self, *input, **kwargs):
```

Comment:
What is the goal of this change of signature?

I actually _liked_ the fact that there are no (**kwargs), the precise list of named arguments...

Reply:
`forward` is a polymorphic method keeping its signature throughout the hierarchy

Reply:
I think for "leaf" classes in hierarchy keeping named kwargs is preferrable because it helps users understand what parameters that particular neural module has.
Does this mess with export?